src/lib.rs

```diff
@@ -324,10 +329,16 @@ impl LightningNode {
         todo!()
     }
 
-    // TODO remove unused_variables after breez sdk implementation
-    #[allow(unused_variables)]
     pub fn get_payment_max_routing_fee_mode(&self, amount_sat: u64) -> MaxRoutingFeeMode {
-        todo!()
+        if amount_sat * (MAX_FEE_PERMYRIAD as u64) / 100 < EXEMPT_FEE_MSAT {
```

Comment:
It should be like follows, right? Maybe we can add some unit tests to catch these types of errors.
```suggestion
        if amount_sat * (MAX_FEE_PERMYRIAD as u64) / 10 < EXEMPT_FEE_MSAT {
```

Reply:
Another option to make it more readable:
```suggestion
        if amount_sat * ((MAX_FEE_PERMYRIAD as u64) / 10_000) * 1_000 < EXEMPT_FEE_MSAT {
```

Reply:
Ah yes, forget my latest comment